### PR TITLE
TP #191622: Public apidoc updates

### PR DESCRIPTION
**Pull request by ulidtko**
* Base-fork Slate version update
 * Overall document structure streamlined, ToC nesting reduced
 * Proofreading fixes
 * Broken internal links fixed
 * `op_status` endpoint documented
 * "Error handling" section added
 * "Webhook callbacks" section added, `POST` method and "≠200 means failure" logic specified

## WIP ##
 * Fact-checking the 2 egress flows and discrepancies in them is pending, but deprioritized due to urgency.

### CODE REVIEW

**Comment by ulidtko** [comment]
To all notified Slate people if any: please ignore. I'm just pulling updates to a downstream fork, and GitHub decides your commits here means you're participants.

File: source/includes/_egress.md [hunk 1]
```diff
@@ -340,43 +337,3 @@ message     | String  | **MIGHT NOT BE PRESENT**: Additional status if user is n
...
-### HTTP Request
-
-<aside class="success"><b><code> POST /rpc/mark_operation </code></b></aside>
-
-### Request JSON Fields
-
-Field             |   Type       | Description
------------------ | ------------ | ---------
-op_id             | String       | UUID reference that was returned from the `send_funds_to_email` endpoint.
-accept            | Boolean      | Either if the merchant wants to approve (true) or reject (false) the process.
```
Comment (ulidtko) [inline source/includes/_egress.md:382]: This was moved to `_lifecycle.md`.

**Comment by ulidtko** [comment]
Hey guys, @jproyo @yowlu — the docs with this PR are available nowhere in rendered form. Here's a quick guide to see it in final form locally, for ease of review.

`bundle install` — needed due to updated Ruby deps
`bundle exec middleman serve` or `./slate.sh serve` as you prefer, will make a locally browseable server

Let me know if issues arise. I've already fixed obsolete ruby version on CircleCI.

**Review by jproyo — APPROVED** [review]
LOTM

**Review by yowlu — COMMENTED** [review]
Looks good to me too!

**Pull request merged**